tests/python/mkl/test_mkldnn.py

```diff
@@ -469,10 +469,9 @@ def check_convolution_training(stype):
 
 
 @with_seed()
-@unittest.skip("Flaky test https://github.com/apache/incubator-mxnet/issues/12579")
 def test_Deconvolution():
     def check_Deconvolution_training(stype):
-        for shape in [(3, 3, 10), (3, 3, 10, 10)]:
+        for shape in [(3, 3, 10, 10)]: # testing only 2D for now
```

Comment:
"for now" is ok for now ;-), as I assume 3D test will be reintroduced soon. But usually "for now" means nothing to the comment reader and should be replaced with something more specific.